### ISSUE TITLE
As a customer, I want to be able to see the restaurant's menu.

### ISSUE THREAD

**Issue by zhoumiabc** [zhoumiabc/Tofu-Kitchen-App#2]
With restaurant's menu, I will be able to know what this restaurant serve.

**Comment by zhoumiabc** [comment]
Breaking down:
As a customer, I want to be able to see the ingredients and pictures of each meal on the menu, so I could choose my favorites.

**Comment by zhangda77** [comment]
with the menu, I could be able to pick up my favorite meal, and try others

**Comment by zhoumiabc** [comment]
Yes, I think your idea is great. Some customer will want save their favorite meal on the app, so they will be able to find their favorite quickly next time.

**Comment by shabbirw** [comment]
The menu will be available to customers as soon as they register. It will not be easy to put a picture for every dish on the menu. It will use too many pages and too much space. What we can do is try to place a picture for every section of the menu, like chicken, sea food, lo mein, rice, etc.
